windows/build.cmd

```diff
@@ -10,12 +10,12 @@ vcpkg install --triplet x86-windows-static --recurse^
  libvorbis[core] libsndfile[core] wildmidi[core] libxmp[core]^
  speexdsp[core] mpg123[core] opusfile[core] fluidsynth-easyrpg[core]^
  inih[cpp] lhasa-easyrpg[core]^
- sdl2-image[core] icu-easyrpg[core] nlohmann-json[core] fmt[core]
+ freeimage[core] icu-easyrpg[core] nlohmann-json[core] fmt[core]

```

Comment:
sdl2[core] must be added as a dependency now. before was pulled in via image 😅

Reply:
Makes sense, thanks!